nomad/local/stop_detach.sh

```diff
@@ -2,5 +2,5 @@
 
 set -euo pipefail
 
-killall nomad || true
+sudo killall nomad || true
```

Comment:
I suspect that this might be broken in regards to not working with the sudoers approved commands, but I'm seeing the dump_artifacts throw an exception so it's not getting to this

Reply:
I have a fix for dump artifacts btw, but good call